### PR TITLE
Added Project entry for Mirage

### PR DESCRIPTION
**Pull request by aaronaxvig**
As requested in #675

### CODE REVIEW

File: gatsby/content/projects/clients/mirage.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: Mirage
+categories:
+ - client
+description: A fancy, customizable, and keyboard-operable Matrix chat client for encrypted and decentralized communication.
+author: miruka
+home: 
+language: QML/Python
+license: LGPL-3.0-or-later
+repo: https://github.com/mirukana/mirage
+room:
```
Comment (aaronraimist) [inline gatsby/content/projects/clients/mirage.mdx]:
```suggestion
room: "#mirage-client:matrix.org"
```

**Comment by jaller94** [comment]
Mirage got added in 0a7cfdc381b3a119fb275d66099507b775a2e544 on April 1.
Feel free to update your branch as this includes a better description IMO.

**Comment by benparsons** [comment]
@aaronaxvig could you update this to reflect the existing file, otherwise this PR can be closed.

**Comment by benparsons** [comment]
Merged manually https://github.com/matrix-org/matrix.org/commit/eceffdf276afb6173d4000a8cfc85f04ed11b72f

**Pull request closed without merging**